### PR TITLE
onepad: Rock Candy Nintendo Switch Controller (#3582)

### LINKED ISSUE #3582: Controller Mapping for Rock Candy Nintendo Switch Controller
1.7.0-0 (GIT)

OnePad Plugin does not contain controller mapping for the Rock Candy Wired Nintendo Switch Controllers.
This is the line that makes it work in onepad.ini

`SDL2 = 030000006f0e00008701000011010000,Performance Designed Products Rock Candy Wired Controller for Nintendo Switch,platform:Linux,a:b2,b:b1,x:b3,y:b0,back:b8,start:b9,guide:b13,leftshoulder:b4,rightshoulder:b5,leftstick:b10,rightstick:b11,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:b6,righttrigger:b7,dpup:h0.1,dpleft:h0.8,dpdown:h0.4,dpright:h0.2,`

### PR DESCRIPTION
**Pull request by RedDevilus**
Fixes  #3582

### CODE REVIEW

File: plugins/onepad/res/game_controller_db.txt [hunk 1]
```diff
@@ -335,6 +335,7 @@
...
 030000006f0e0000a802000023020000,PDP Wired Controller for Xbox One,a:b0,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b10,leftshoulder:b4,leftstick:b11,lefttrigger:b6,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b12,righttrigger:b7,rightx:a2,righty:a3,start:b9,x:b2,y:b3,platform:Linux,
 030000006f0e00008501000011010000,PDP Wired Fight Pad Pro for Nintendo Switch,a:b2,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b12,leftshoulder:b4,leftstick:b10,lefttrigger:b6,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b11,righttrigger:b7,rightx:a2,righty:a3,start:b9,x:b3,y:b0,platform:Linux,
+030000006f0e00008701000011010000,Performance Designed Products Rock Candy Wired Controller for Nintendo Switch,a:b2,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b13,leftshoulder:b4,leftstick:b10,lefttrigger:b6,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b11,righttrigger:b7,rightx:a2,righty:a3,start:b9,x:b3,y:b0,platform:Linux,
```
Comment (RedDevilus) [inline plugins/onepad/res/game_controller_db.txt:338]: Performance Designed Products
to
PDP
?

Comment (lightningterror) [reply plugins/onepad/res/game_controller_db.txt:338]: `PDP Rock Candy Wired Controller for Nintendo Switch` I guess?

Comment (RedDevilus) [reply plugins/onepad/res/game_controller_db.txt:338]: Yeah i wasn't sure if you could change the name. Thanks

File: plugins/onepad/res/game_controller_db.txt [hunk 2]
```diff
@@ -335,6 +335,7 @@
 030000006f0e00000901000011010000,PDP Versus Fighting Pad,a:b1,b:b2,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b12,leftshoulder:b4,lefttrigger:b6,rightshoulder:b5,righttrigger:b7,start:b9,x:b0,y:b3,platform:Linux,
 030000006f0e0000a802000023020000,PDP Wired Controller for Xbox One,a:b0,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b10,leftshoulder:b4,leftstick:b11,lefttrigger:b6,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b12,righttrigger:b7,rightx:a2,righty:a3,start:b9,x:b2,y:b3,platform:Linux,
 030000006f0e00008501000011010000,PDP Wired Fight Pad Pro for Nintendo Switch,a:b2,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b12,leftshoulder:b4,leftstick:b10,lefttrigger:b6,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b11,righttrigger:b7,rightx:a2,righty:a3,start:b9,x:b3,y:b0,platform:Linux,
+030000006f0e00008701000011010000,PDP Rock Candy Wired Controller for Nintendo Switch,a:b2,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b13,leftshoulder:b4,leftstick:b10,lefttrigger:b6,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b11,righttrigger:b7,rightx:a2,righty:a3,start:b9,x:b3,y:b0,platform:Linux,
```
Comment (lightningterror) [inline plugins/onepad/res/game_controller_db.txt:338]: Maybe move it below PDP Kingdom Hearts controller? Alphabetical order.

**Pull request merged**